pygments/lexers/tal.py

```diff
@@ -0,0 +1,68 @@
+"""
+    pygments.lexers.tal
+    ~~~~~~~~~~~~~~~~~~~
+
+    Lexer for Uxntal
+
+    .. versionadded:: 2.12
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import RegexLexer, words
+from pygments.token import Comment, Keyword, Name, String, Number, Punctuation, Whitespace, Literal
+
+__all__ = ['TalLexer']
+
+class TalLexer(RegexLexer):
```

Comment:
Oh, and the warning when building the docs on your branch reminds me that the lexer class needs a docstring, for https://pygments.org/docs/lexers/. Look at existing docstrings for examples (and don't forget the `.. versionadded::`).

Reply:
Added!